geocat/comp/meteorology.py

```diff
@@ -219,9 +219,9 @@ def _relhum(
     """Calculates relative humidity with respect to ice, given temperature,
```

Comment:
```suggestion
    """Calculates relative humidity given temperature,
```

I think this was a copy-paste error perhaps.

From the [NCL docs](https://www.ncl.ucar.edu/Document/Functions/Built-in/relhum.shtml):

> Calculates relative humidity given temperature, mixing ratio, and pressure, using a table look-up procedure. The algorithm and tables can yield some very high relative humidities (> 100%) at high pressures and very low temperature.
> 
> Relative Humidity between 0C and -20C are calculated with respect to a mixed phase between ice and water. Below -20C the returned humidities are calculated with respect to ice. Below -20C the results will be very similar to those from [relhum_ice](https://www.ncl.ucar.edu/Document/Functions/Built-in/relhum_ice.shtml) and above 0C the results will be very similar to [relhum_water](https://www.ncl.ucar.edu/Document/Functions/Built-in/relhum_water.shtml).